src/commands/method/send-transaction.ts

```diff
@@ -1,8 +1,9 @@
-const Base = require('../../base')
-const { cli } = require('cli-ux')
-const { getNetworkFlags } = require('../../helpers/networks')
+import { cli } from 'cli-ux'
 
-class SendTransactionCommand extends Base {
+import { BaseCommand } from '../../base'
+import { getNetworkFlags } from '../../helpers/networks'
+
```

Comment:
Q: Can we move the `getNetworkFlags` as a property in the abstract class `BaseCommand`? I think could we do it because now we have typescript (should...) (see [docs](https://oclif.io/docs/base_class) ), or do you consider it better to do it in another PR?

Reply:
I started working on that but then I ran into an issue. It's reported here: https://github.com/oclif/oclif/issues/225